index.js

```diff
@@ -27,22 +27,22 @@ exports.search = function (query, options, callback) {
 }
 
 function search (query, options, callback) {
-  var page = options.page || 1
-  return request({
-    method: 'get',
-    url: 'https://www.npmjs.com/search?page=' + page + '&q=' + query
-  }, function (err, res, body) {
-    if (err) return callback(err)
-    var $ = cheerio.load(body)
-    var modules = []
-    $('.package-details').each(function () {
-      modules.push({
-        name: $(this).find('.packageName').text(),
-        version: $(this).find('.version').text(),
-        author: $(this).find('.authorName').text(),
-        description: $(this).find('.description').text()
-      })
+    var page = options.page - 1 || 0
+    return request({
+        method: 'get',
+        url: 'https://www.npmjs.com/search?page=' + page + '&q=' + query
+    }, function(err, res, body) {
+        if (err) return callback(err)
+        var $ = cheerio.load(body)
+        var modules = []
+        $('.package-list-item__capsule___3_4Eo').each(function() {
+            modules.push({
+                name: $(this).find('.package-list-item__title___sqwj8').text(),
```

Comment:
Oh, they probably use react or sheetify that generates random variables. 
This won't work as it could change at any time.

We'll need to find another solution.